### DISCUSSION TITLE
List Issued Documents - Filtro documenti pagati/non pagati

### DISCUSSION THREAD

**Discussion by aikonfuture** [fattureincloud/api#80] [Q&A] [answered]
Buongiorno,
nelle api v1.0 per la chiamata **/{tipo-doc}/lista** esisteva il parametro **saldato** 

> (boolean, opzionale): Elenca solo i documenti saldati completamente (true) o non saldati/saldati parzialmente (false)

Con la v2 per la chiamata **/c/{company_id}/issued_documents** non mi sembra ci sia una proprietà simile. Come occorre impostare la query string per ottenere lo stesso risultato ? Quale valore dovrei filtrare con il parametro 'q' ? 

Grazie.
Saluti.

**Comment by emajo** [comment, marked as answer]
Ciao @aikonfuture per filtrare i documenti pagati il filtro è il seguente:
`q=next_due_date is not null`

Resto a disposizione per ulteriori chiarimenti,
Emanuele

**Reply by aikonfuture** [reply]
Grazie Emanuale.
Funzionerebbe anche nel seguente scenario ? 
Es. fattura 100€ pagamento singola scadenza a 30 giorni. Viene effettuato un pagamento parziale (capita purtroppo). Cosa succede a next_due_date ? Rimane valorizzato al valore originario ?

**Reply by emajo** [reply]
Nel caso in cui `fattura 100€ con singola scadenza` (quindi un pagamento singolo nei `payments_list`) supponiamo la divisione in due tranches di pagamenti, una da 60€ e una da 40€, deve essere modificato il documento aggiustando i `payments_list`, mettiamo caso che venga poi saldata la tranche da 60€, il payment item corrispondente dovrà essere messo a `status = paid` e in `next_due_date` ci sarà quindi la `due_date` della tranche di 40€ rimanente.
